source/code-snippets/crud/cursor.js

```diff
@@ -60,6 +60,16 @@ async function rewind(myColl) {
   // end rewind cursor example
 }
 
+async function listIndexSearch(myColl){
```

Comment:
See example code [here](https://github.com/10gen/cloud-docs/blob/master/source/includes/fts-tutorial/search-index-management/list-indexes.js)

Also, if you move the content from the Cursor page to the indexes page, you will have to move this code to a different file as well.